iothub/service/src/DigitalTwin/Customized/DigitalTwinInvokeCommandHeaders.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.Devices
+{
+    /// <summary>
+    /// Defines headers for InvokeCommandAsync and InvokeComponentCommandAsync operation.
+    /// </summary>
+    public class DigitalTwinInvokeCommandHeaders
+    {
+
+        /// <summary>
+        /// Initializes a new instance of the
+        /// DigitalTwinInvokeCommandHeaders class.
+        /// </summary>
+        /// <param name="requestId">Server Generated Request Id (GUID), to uniquely identify this request in the service</param>
+        public DigitalTwinInvokeCommandHeaders(string requestId = default)
```

Comment:
so the default is null, what happens if we send null as the request Id ? does the service assign a request Id value by default?

Reply:
So the request id is something that the server generates and returns to us as part of the response. In the CL, we grab that request id from response.Headers.XMsRequestId and assign it to the property RequestId in this class. So this will be whatever the server returns to us.